### PR TITLE
added get list transfer and transfer methods

### PR DESCRIPTION
**Pull request by madhankumar3103**
added get  transfer and list transfer and list transfer methods

### CODE REVIEW

**Review by arao6 — CHANGES_REQUESTED** [review]
Please fix the lint errors

**Review by Rrathinasabapath — APPROVED** [review]
Approved. Ready for others to review.

File: hyperwallet/tests/test_encryption.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def test_should_fail_decryption_when_wrong_private_key_is_used(self):
         with self.assertRaises(HyperwalletException) as exc:
             encryption2.decrypt(encryptedMessage)
 
-        self.assertTrue(str(exc.exception).startswith('No recipient matched the provided key["Failed: [ValueError(\'Decryption failed.\''))
```
Comment (vkorotchenko) [inline hyperwallet/tests/test_encryption.py:44]: did the expected behavior of this case change? / are we not expecting to see the Failed value anymore?

Comment (madhankumar3103) [reply hyperwallet/tests/test_encryption.py:44]: Actually this is for old test case ,i did not modifying any code for encryption but i am facing build error so i modifying this line the build got success.

**Review by vkorotchenko — APPROVED** [review]

**Review by arao6 — APPROVED** [review]

**Pull request merged**